ui/app/templates/rules.html

```diff
@@ -205,6 +205,11 @@ <h5 title="Fallback Mapping" ng-show="rule.fallbackMapping || (show_sc && rule.s
             </a>
           </span>
       </h5>
+      <span ng-show="(rule.fallbackMapping || (!rule_id && show_sc && rule.scheduled_change !== null && rule.scheduled_change.fallbackMapping !== null )) && (rule.backgroundRate === 100 || rule.scheduled_change.backgroundRate === 100)" >
```

Comment:
Closer, but this still isn't working in at least a couple of cases. For example, the note should be shown in this case: https://screenshots.firefoxusercontent.com/images/92e2a8b5-8969-4234-a9e1-32787b97daf7.png

Part of the problem is that the last part of this condition uses "=== 100" -- that's actually the opposite of what we want, it should _never_ unless the backgroundRate of the Rule or Scheduled Change is _not_ 100.

Please make sure you do some of your own testing locally to ensure the cases I identified earlier are working.

Reply:
This was difficult to digest for me, until I went through the bug again, and realized why this is so. Actually I took it other way as if fallback Mapping is already set, but here we  are trying to set it afterwards. 

I tested all cases and it should work now.